tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary Section Status.feature

```diff
@@ -181,7 +181,7 @@ Scenario: the order section status is set for an imcomplete order with one solut
         | 10001   | 1000-001        | ODS1    |
     When the user makes a request to retrieve the order summary with the ID 10001
     Then a response with status code 200 is returned
-    And the order Section Status is incomplete
+    And the order Section Status is complete
```

Comment:
This scenario is incorrect now. When there is a catalogue solution in the order, the order section status should only be complete once the additional services and associated services sections have also been viewed. Therefore this change will need to be reverted.

Reply:
Revert is done